zerotwobot/modules/telegraph.py

```diff
@@ -1,92 +1,85 @@
-"""
-Telegraph module to upload text and image/video to telegra.ph via zerotwobot
-Author/Written - @kishoreee
-Copy with credits else face legal problems.
-"""
-
 import os
 from datetime import datetime
+from telethon import events
 
 from PIL import Image
-from telegraph.aio import Telegraph
-
-from zerotwobot import TEMP_DOWNLOAD_LOC, application
-from telegram import Update
-from telegram.ext import ContextTypes, CommandHandler
-
+from telegraph import Telegraph, exceptions, upload_file
 
-async def telegraph(update: Update, context: ContextTypes.DEFAULT_TYPE):
+from zerotwobot import telethn, TEMP_DOWNLOAD_LOC
 
-    telegrph = Telegraph()
 
-    r = await telegrph.create_account(short_name="zerotwobot")
-    auth_url = r["auth_url"]
+telegraph = Telegraph()
+r = telegraph.create_account(short_name="ZeroTwo")
```

Comment:
`requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))`

Maybe we can override the library to use an alternative URL or fork the library and change url.
I don't know how it's working for you, but if it doesn't work on my machine, then it will probably fail in my server too.
So can't take risk on this one.